composer.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "platformsh/template-wordpress",
+  "description": "WordPress Composer-ified. A template for managing WordPress core, plugins, and themes.",
+  "keywords": [
+    "wordpress",
+    "blog",
+    "cms",
+    "template"
+  ],
+  "homepage": "https://github.com/platformsh/template-wordpress",
```

Comment:
```suggestion
  "homepage": "https://github.com/platformsh-templates/wordpress",
```

Reply:
I didn't think to look for this before. This is really only important for linking when these are published, yes? So maybe not that important right now.